src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -48,45 +46,6 @@ public void init() {
     datastore = new Datastore();
   }
 
-  /**
-   * Responds with a JSON representation of {@link Message} data for a specific user. Responds with
-   * an empty array if the user is not provided.
-   */
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Why are we removing this?

Reply:
We can get user's list message from server using jsp file from now on, so we don't need to fetch messages using MessageServlet anymore

Reply:
If we hadn't removed that, the code would be duplicated